controllers/datadogagent/feature/dummy/feature.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package dummy
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/DataDog/datadog-operator/apis/datadoghq/v1alpha1"
+	"github.com/DataDog/datadog-operator/apis/datadoghq/v2alpha1"
+	"github.com/DataDog/datadog-operator/controllers/datadogagent/feature"
+)
+
+func init() {
+	err := feature.Register(feature.DummyIDType, buildDummyfeature)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func buildDummyfeature(options *feature.Options) feature.Feature {
+	return &dummyFeature{}
+}
+
+const (
+	dummyLabelKey   = "feature.com/dummy"
+	dummyLabelValue = "true"
+)
+
+type dummyFeature struct{}
+
+func (f *dummyFeature) Configure(dda *v2alpha1.DatadogAgent) bool {
+	return false
+}
+
+func (f *dummyFeature) ConfigureV1(dda *v1alpha1.DatadogAgent) bool {
```

Comment:
Should not be necessary with conversion in place

Reply:
Agree, But for now. I use it to validate the PR in https://github.com/DataDog/datadog-operator/pull/430

we can easily remove it later